src/Applications/LDAS_App/lenkf.j.template

```diff
@@ -357,10 +357,13 @@ while ( $counter <= ${NUM_SGMT} )
    
    if( -e EGRESS.ldas ) then
       set rc = 0
+      echo GEOSldas Run Status: $rc
    else
       set rc = -1
+      echo GEOSldas Run Status: $rc
+      echo "GEOSldas run failed, exit without post-processing"
+      break
```

Comment:
If we "break" here, don't we then immediately hit the "exit" statement in line 740? If this is the desired behavior, then we might as well "exit" right here, or not? 
My earlier thought was that we should try to finish out the current SGMT, and then have the if block with the rc-triggered exit in ~line 714, but on 2nd thought I'm not sure this makes much sense.
In summary, change "break" back to "exit" here and remove the new if block that was added ~Line 738?